Examples/Swift/ViewController.swift

```diff
@@ -19,15 +19,16 @@ class ViewController: UIViewController, MGLMapViewDelegate {
     @IBOutlet weak var toggleNavigationButton: UIButton!
     @IBOutlet weak var howToBeginLabel: UILabel!
     
-    // `identityPoolId` is a required value for using AWS Polly voice instead of iOS's built in AVSpeechSynthesizer
-    // You can get a token here: http://docs.aws.amazon.com/mobile/sdkforios/developerguide/cognito-auth-aws-identity-for-ios.html
-    var routeVoiceController = RouteVoiceController(identityPoolId: "<#Your AWS IdentityPoolId. Remove Argument if you do not want to use AWS Polly#>")
+    var routeVoiceController = RouteVoiceController()
```

Comment:
Oh, this can be removed. It's just that the example tries to show how to do this w/ _and_ w/o NavigationUI.